### ISSUE TITLE
SohoModuleNav: Routing does not work when module nav is collapsed.

### ISSUE THREAD

**Issue by Abrai7** [infor-design/enterprise-ng#1674]
**Describe the bug**
When the module-nav component is set to collapsed mode any <a> tag doesn't redirect the user to the correct path. 
This will be solved after expanding the component and collapsing it again.

**To Reproduce**
1. Go to https://ids-quick-start-1711-jtxbae.stackblitz.io
2. Click on 'B' item. (Issue is observed)
3. Click on 'Em' Icon (Observe the module-nav is now expanded).
4. Click on 'B' item (Redirection is triggered)

**Expected behavior**
Routing event should work on the first try.
**Version**
- ids-enterprise-ng: 4.86

**Screenshots**
<img width="230" alt="image" src="https://github.com/infor-design/enterprise-ng/assets/9669865/da43c75e-86d4-4a0a-884f-2977969baa11">

<img width="228" alt="image" src="https://github.com/infor-design/enterprise-ng/assets/9669865/71477831-0660-4651-9eb5-4b83d71f2e02">

**Platform**
 - Browser Name: Chrome
 - Browser Version: Version 123.0.6312.106 (Official Build) (64-bit)

**Comment by tmcconechy** [comment]
We can look into this but for new adoption you might consider the web component version as its a lot cleaner (did not test if it has this bug)

**Comment by tmcconechy** [comment]
No, Its still in the backlog will try to get it in next few sprints

**Comment by eoulman99** [comment]
@tmcconechy any chance this can be bumped up?  Multiple customers reporting this case

**Comment by tmcconechy** [comment]
Its now in progress https://inforwiki.atlassian.net/browse/IDS-4185 will post updates when it is resolved.

**Comment by tmcconechy** [comment]
See notes on https://inforwiki.atlassian.net/browse/IDS-4185 -> we could not reproduce this.
